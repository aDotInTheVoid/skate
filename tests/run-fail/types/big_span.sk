fn main() {
    let x = {
        {
            {
                2
            }
        }
    } + {
        {
            {
                null
            }
        }
    }
}