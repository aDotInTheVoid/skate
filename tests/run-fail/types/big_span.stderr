error: Unknown binop `+`, for `int` and `null`
   ┌─ $DD/tests/run-fail/types/big_span.sk:8:7
   │    
 2 │         let x = {
   │ ╭───────────────'
 3 │ │           {
 4 │ │               {
 5 │ │                   2
 6 │ │               }
 7 │ │           }
 8 │ │       } + {
   │ │         ^ In this operator
   │ ╰───────' LHS evaluated to Int(2)
   │   ╭─────────'
 9 │   │         {
10 │   │             {
11 │   │                 null
12 │   │             }
13 │   │         }
14 │   │     }
   │   ╰─────' LHS evaluated to Null

